alb.tf

```diff
@@ -38,6 +38,18 @@ resource "aws_alb_target_group" "main" {
     }
   }
 
+  dynamic "stickiness" {
+    for_each = length(keys(lookup(var.target_groups[count.index], "stickiness", {}))) == 0 ? [] : [
+    lookup(var.target_groups[count.index], "stickiness", {})]
+
+    content {
+      cookie_duration = lookup(stickiness.value, "cookie_duration", null)
+      cookie_name     = lookup(stickiness.value, "cookie_name", null)
+      enabled         = lookup(stickiness.value, "enabled", null)
```

Comment:
maybe use the default `true` here from https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lb_target_group#stickiness?

Reply:
i have included the default value to set it to true.